vault/resource_secrets_sync_aws_destination_test.go

```diff
@@ -78,6 +82,8 @@ resource "vault_secrets_sync_aws_destination" "test" {
   access_key_id        = "%s"
   secret_access_key    = "%s"
   region               = "%s"
+  role_arn			   = "role-arn-test"
```

Comment:
nit: formatting

Reply:
Ah, not sure what's going on here. I ran the formatting command, but doesn't apply any changes and it looks right on my side.